readthedocs/projects/tasks.py

```diff
@@ -1899,15 +1900,16 @@ def send_build_status(build_pk, commit, status, link_to_build=False):
     if provider_name in [GITHUB_BRAND, GITLAB_BRAND]:
         # get the service class for the project e.g: GitHubService.
         service_class = build.project.git_service_class()
-        users = build.project.users.all()
```

Comment:
If the `try` block does not return or raise an exception then the `except` block won't be executed and Local variable 'users' will be referenced before assignment here

https://github.com/readthedocs/readthedocs.org/blob/be8282ad75e8f9632a2b055ec6ba00940566e6fb/readthedocs/projects/tasks.py#L1965

Reply:
Thanks! I shouldn't have changed that :)

Moving it back where it was.